src/Sidebar/ui/SidebarRoot.tsx

```diff
@@ -160,6 +163,24 @@ export function SidebarRoot(props: IProps): React.ReactElement {
   const canBladeburner = !!Player.bladeburner;
   const canStaneksGift = Player.augmentations.some((aug) => aug.name === AugmentationName.StaneksGift1);
 
+  const clickPage = useCallback(
+    (page: Page) => {
+      if (page === Page.Job) {
+        Router.toJob(Locations[Object.keys(Player.jobs)[0]]);
+      } else if (page == Page.ScriptEditor) {
+        Router.toScriptEditor();
+      } else if ((Object.values(SimplePage) as Page[]).includes(page)) {
+        Router.toPage(page as SimplePage);
```

Comment:
There's an "isSimplePage" function for this check, which also removes the need to do page `as` SimplePage